README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python?ref=badge_large)
 
-test
+test2
```

Comment:
General comment: The changes made to the README.md file seem to be test or placeholder content. It's important to keep the README professional and informative. Please review the entire README to ensure it accurately represents the project and provides valuable information to users and contributors.